raster_download/src/main/java/de/meggsimum/sauber/sdi/RasterDownloader.java

```diff
@@ -94,12 +97,15 @@ public RasterDownloader() throws Exception {
 	 * 
 	 * @throws Exception
 	 */
+
+	 
 	public void resolveSecrets() throws Exception {
+				
 		String workingDir = System.getProperty("user.dir");
-		File secretFileUsrCtn = new File("/run/secrets/hhi-rest-user.txt");
-		File secretFileUsrLoc = new File(workingDir + "/../secrets/hhi-rest-user.txt");
-
+		
 		// load HHI REST user from docker secret or use local dev file as backup
+		File secretFileUsrCtn = new File("/run/secrets/hhi_rest_user");

```

Comment:
Please rename var Since we have more than one user now.
```suggestion
		File secretFileHhiRestUsrCtn = new File("/run/secrets/hhi_rest_user");

```